modules/kubernetes/kube-state-metrics/README.md

```diff
@@ -30,28 +30,29 @@ Handles discovery of kubernetes targets and exports them, this component does no
+| Label     | Description                                                                                       |
+| :-------- | :------------------------------------------------------------------------------------------------ |
+| `source`  | Constant value of `kubernetes`, denoting where the results came from, this can be useful for LBAC |
 
 ---
 
 ### `scrape`
 
 #### Arguments
 
-| Name              | Required | Default                                      | Description                                                                                                                                         |
-| :---------------- | :------- | :------------------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `targets`         | _yes_    | `list(map(string))`                          | List of targets to scrape                                                                                                                           |
-| `forward_to`      | _yes_    | `list(MetricsReceiver)`                      | Must be a where scraped should be forwarded to                                                                                                      |
-| `job_label`       | _no_     | `integrations/kubernetes/kube-state-metrics` | The job label to add for all metrics                                                                                                                |
-| `keep_metrics`    | _no_     | [see code](module.river#L228)                | A regular expression of metrics to keep                                                                                                             |
-| `drop_metrics`    | _no_     | [see code](module.river#L235)                | A regular expression of metrics to drop                                                                                                             |
-| `scrape_interval` | _no_     | `60s`                                        | How often to scrape metrics from the targets                                                                                                        |
-| `scrape_timeout`  | _no_     | `10s`                                        | How long before a scrape times out                                                                                                                  |
-| `max_cache_size`  | _no_     | `100000`                                     | The maximum number of elements to hold in the relabeling cache.  This should be at least 2x-5x your largest scrape target or samples appended rate. |
-| `clustering`      | _no_     | `false`                                      | Whether or not [clustering](https://grafana.com/docs/kube-state-metrics/latest/flow/concepts/clustering/) should be enabled                         |
```

Comment:
Most of this is reformatting.
`scheme` and `bearer_token_file` were added.
`keep_metrics` and `drop_metrics` had their links updated.